code/game/objects/effects/landmarks/excavation_site_spawner.dm

```diff
@@ -0,0 +1,21 @@
+/obj/effect/landmark/excavation_site_spawner
```

Comment:
i absolutly don't get why you would need a landmark spawner and a landmark site.

You could have done everything with just one landmark type

Reply:
changed to use only the spawner